packet/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -42,30 +42,10 @@ resource "packet_device" "controllers" {
   hostname         = "${var.cluster_name}-controller-${count.index}"
   plan             = "${var.controller_type}"
   facilities       = ["${var.facility}"]
-  operating_system = "custom_ipxe"
+  operating_system = "flatcar_${var.os_channel}"
   billing_cycle    = "hourly"
   project_id       = "${var.project_id}"
-  ipxe_script_url  = "${var.ipxe_script_url}"
-  always_pxe       = "false"
-  user_data        = "${element(data.ct_config.controller-install-ignitions.*.rendered, count.index)}"
-}
-
-data "ct_config" "controller-install-ignitions" {
-  count   = "${var.controller_count}"
-  content = "${element(data.template_file.controller-install.*.rendered, count.index)}"
-}
-
-data "template_file" "controller-install" {
-  count    = "${var.controller_count}"
-  template = "${file("${path.module}/cl/controller-install.yaml.tmpl")}"
-
-  vars {
-    os_channel           = "${var.os_channel}"
-    os_version           = "${var.os_version}"
-    flatcar_linux_oem    = "packet"
```

Comment:
Note to self: When testing this, see if the packet OEM is set, so it uses the bond for network and out of band console special kernel params

Reply:
Bond network and serial console both seem fine on my testing. I am pretty sure this is set by the Packet installation procedure.